src/util/ThreadPool.h

```diff
@@ -83,9 +83,9 @@ class CommandForThreadPool : public CommandTracked {
 public:
     using Ptr = std::shared_ptr<CommandForThreadPool>;
 
-    CommandForThreadPool() = default;
+    CommandForThreadPool() : CommandTracked() {}
```

Comment:
What's the reason for this change? It's the same as `= default`

Reply:
Same as the similar comment before.